### PR TITLE
Rename `ManaStructure` to `ManaParameters`

### LINKED ISSUE #1434: Rename ManaStructure to ManaParameters

### PR DESCRIPTION
**Pull request by Alex6323**
## Links to any relevant issues

Closes #1434

### CODE REVIEW

File: sdk/src/types/block/mana/structure.rs
Comment (thibault-martinez) [inline sdk/src/types/block/mana/structure.rs:1]: Rename the file

File: sdk/src/types/block/protocol.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub struct ProtocolParameters {
     pub(crate) slots_per_epoch_exponent: u8,
     /// The parameters used for mana calculations.
     #[getset(skip)]
-    pub(crate) mana_structure: ManaStructure,
+    pub(crate) mana_structure: ManaParameters,
```
Comment (thibault-martinez) [inline sdk/src/types/block/protocol.rs]: rename

File: sdk/src/types/block/mana/parameters.rs [hunk 2]
```diff
@@ -255,7 +255,7 @@ mod test {
 
     #[test]
     fn test_mana_decay_no_factors() {
-        let mana_structure = ManaStructure {
+        let mana_structure = ManaParameters {
```
Comment (thibault-martinez) [inline sdk/src/types/block/mana/parameters.rs]: rename

Comment (thibault-martinez) [reply sdk/src/types/block/mana/parameters.rs]: And please do a search in the whole repo for `mana_structure`

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**